osu.Game.Rulesets.Taiko/UI/TaikoPlayfieldAdjustmentContainer.cs

```diff
@@ -12,11 +13,22 @@ public partial class TaikoPlayfieldAdjustmentContainer : PlayfieldAdjustmentCont
     {
         private const float default_relative_height = TaikoPlayfield.DEFAULT_HEIGHT / 768;
 
+        public const float DEFAULT_ASPECT = 16f / 9f;
         public const float MAXIMUM_ASPECT = 16f / 9f;
         public const float MINIMUM_ASPECT = 5f / 4f;
 
         public readonly IBindable<bool> LockPlayfieldAspectRange = new BindableBool(true);
 
+        public static double AspectRatioToTimeRange(float aspectRatio)
+        {
+            const float default_time_range = 7000 / TaikoBeatmapConverter.VELOCITY_MULTIPLIER;
+
+            // This is the fraction of the width that should not contribute to time range.
+            const float ratio_compensator = 380f / 1366f;
```

Comment:
Would name it `non_playable_portion` at least.

Reply:
Renamed in a46ca142886163eb839b4781b0fc929e5a946801